src/components/ArticlesCard.jsx

```diff
@@ -0,0 +1,28 @@
+import {useState} from 'react'
+import styles from '../css/ArticlesCard.module.css'
+<css />
+
+function ArticlesCard({allArticles}){
+
+
+
+    return(<>
+    <section >
+    {allArticles.map((article)=>{
+            return <li key={article.article_id} className={styles.card}>
+                <h2>{article.title}</h2>
+                <p><span className={styles.cardspan}>Created by: </span>{article.author}</p>
+                <p>{article.created_at}</p>
```

Comment:
We also want to see when the articles were created in a human readable format as they should already be sorted by date when they come back from the api as well as the comment count